wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -185,5 +185,12 @@
   /** When generating workspace name from generateName, append this many characters. */
   public static final int WORKSPACE_GENERATE_NAME_CHARS_APPEND = 5;
 
+  /**
+   * The attribute in the workspace config where we store the infrastructure namespace the workspace
```

Comment:
Using `workspace config` here is confusing because it's actually supposed to be store in `Workspace#attributes` but not in `WorkspaceConfig#attributes`.
https://github.com/eclipse/che/blob/1663ab5a8bc55ce1696d6063d2c7e3293a32f4f7/infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesNamespaceFactory.java#L427
Please consider removing config word here or even better may be referencing model interface.

Reply:
Indeed. Thanks for spotting this!